Sources/NIO/ChannelPipeline.swift

```diff
@@ -900,29 +877,49 @@ extension ChannelPipeline {
     ///
     /// - parameters:
     ///     - handlers: The array of `ChannelHandler`s to be added.
-    ///     - first: If `true`, the supplied `ChannelHandler`s will be added to the front of the pipeline.
-    ///              If `false`, they will be added to the back.
+    ///     - position: The position in the `ChannelPipeline` where to add `handlers`, defaults to `.last`.
```

Comment:
Nit: "The position in the `ChannelPipeline` to add `handlers`. Defaults to `.last`."